vitrina/celery.py

```diff
@@ -1,10 +1,73 @@
 import os
+from typing import Any
 
 from celery import Celery
+import reversion
+from django.contrib.auth import get_user_model
+from vitrina.utils import RevisionComment, RevisionSource
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "vitrina.settings")
 app = Celery("vitrina")
 app.config_from_object("django.conf:settings", namespace="CELERY")
-app.autodiscover_tasks()
 
 app.conf.timezone = "UTC"
+
+
+class RevisionedTask(app.Task):
+    """Celery task base class that records Django Reversion revisions for each run.
+
+    Responsibilities:
+    - Propagates a `_reversion_user_id` (if provided) into task message headers for downstream use.
+    - Wraps task execution in a Reversion revision, attaching the user (when resolvable)
+      and a structured JSON comment describing the task call.
+    """
+
+    abstract = True
+
+    def apply_async(self, args: Any | None = None, kwargs: Any | None = None, **options) -> Any:
```

Comment:
`None` is a part of `Any` already: 

```suggestion
    def apply_async(self, args: Any | None = None, kwargs: Any = None, **options) -> Any:
```